spores-pickling/src/test/scala/scala/spores/run/pickling/PicklingBinary.scala

```diff
@@ -20,63 +27,66 @@ trait Emitter[T] {
 
 class TestEmitter extends Emitter[String] {
   val builder = new StringBuilder()
-  def emit(v: String)(implicit pickler: Pickler[String], unpickler: Unpickler[String]): Unit =
+
+  def emit(v: String)(implicit pickler: Pickler[String],
+                               unpickler: Unpickler[String]): Unit = 
     builder.append(v)
+
   def done(): Unit = {
     // do nothing
   }
+
 }
 
 @RunWith(classOf[JUnit4])
 class PicklingBinarySpec {
+
+  import static._
+
   @Test
   def `pickle/unpickle to/from binary`(): Unit = {
+
     val v1 = 10
     val s = spore {
       val c1 = v1
       (x: Int) => s"arg: $x, c1: $c1"
     }
 
-    val pickler: Pickler[Spore[Int, String] { type Captured = Int }] with Unpickler[Spore[Int, String] { type Captured = Int }] =
-    SporePickler.genSporePickler[Int, String, Int]
-
-    val format = implicitly[PickleFormat]
-    val builder = format.createBuilder
-
-    builder.hintTag(implicitly[FastTypeTag[Spore[Int, String]]])
-    pickler.pickle(s, builder)
-    val res = builder.result()
-
-    val reader = format.createReader(res.asInstanceOf[format.PickleType])
-    val up = pickler.unpickle("scala.spores.Spore[Int, String]", reader)
-    val us = up.asInstanceOf[Spore[Int, String]]
-    val res2 = us(5)
+    val res = s.pickle
+    val up = res.value.unpickle[Spore[Int, String]]
+    val res2 = up(5)
     assert(res2 == "arg: 5, c1: 10")
+
   }
 
   @Test
   def `generated nested classes`(): Unit = {
+
     val s = spore {
       implicit val p = implicitly[Pickler[(Int, List[String])]]
       implicit val u = implicitly[Unpickler[(Int, List[String])]]
       (elem: (Int, List[String]), emit: Emitter[(Int, List[String])]) =>
         if (elem._1 == 0) emit.emit(elem)
     }
+
     assert(true)
+
   }
 
   @Test
   def testSimpleSpore(): Unit = {
+
```

Comment:
Btw, normally we don't have blank lines at the beginning of method bodies. So, I'm a bit reluctant to accept introducing this "convention" into existing code. I think new code should have a style similar to the existing code. This would also be consistent with Scala's code base. OK?